staging/src/k8s.io/client-go/util/flowcontrol/backoff_test.go

```diff
@@ -27,15 +27,27 @@ func TestSlowBackoff(t *testing.T) {
 	id := "_idSlow"
 	tc := clock.NewFakeClock(time.Now())
 	step := time.Second
-	maxDuration := 50 * step
+	maxDuration := 50 * time.Second
 
 	b := NewFakeBackOff(step, maxDuration, tc)
-	cases := []time.Duration{0, 1, 2, 4, 8, 16, 32, 50, 50, 50}
-	for ix, c := range cases {
+	cases := []time.Duration{
+		0,
+		time.Second,
+		time.Duration(1821921541),
```

Comment:
I greatly appreciate the test, but this is kind of a change detector right now, and it'll be awkward to update if we e.g. change the ratio (as I'm sure you must already appreciate, having somehow gotten the correct batch of random numbers in here).

I suggest instead
* make jitter a field so you can vary it; set it to 0 for these tests, so that they continue to test the exponential math exactly
* make a new test that proves that the jittered time is always in the expected bounds
* make a test that proves that jitter + exponential backoff stays within the expected bounds